### PR TITLE
Add helm fixes

### PR DESCRIPTION
**Pull request by iblxslav**
- add helm fixes
- make sure we run `go mod tidy`
- use docker buildx for multi-arch builds

### CODE REVIEW

File: atlas/commands/bootstrap/bootstrap.go [hunk 1]
```diff
@@ -181,6 +181,9 @@ func generateProtobuf() error {
 // initGoMod calls "go mod init" to generate .toml files
 func initGoMod() error {
 	fmt.Print("Starting mod project... ")
+	if err := runCommand("go", "mod", "tidy"); err != nil {
+		return err
+	}
 	if err := runCommand("go", "mod", "vendor"); err != nil {
 		return err
 	}
```
Comment (kd7lxl) [inline atlas/commands/bootstrap/bootstrap.go:189]: There's no need for `go mod download` after `go mod vendor`.

Comment (iblxslav) [reply atlas/commands/bootstrap/bootstrap.go:189]: removed

File: atlas/templates/Makefile.common.gotmpl [hunk 2]
```diff
@@ -77,7 +77,7 @@ test-atlas: fmt-atlas
 	$(GO_RUNNER) go test $(GO_TEST_FLAGS) $(GO_PACKAGES)
 
 docker-atlas:
-	@docker build -f $(SERVER_DOCKERFILE) -t $(SERVER_IMAGE):$(IMAGE_VERSION) .
+	@docker buildx build -f $(SERVER_DOCKERFILE) -t $(SERVER_IMAGE):$(IMAGE_VERSION) .
```
[the remark this answers is not in the document]
Comment (iblxslav) [reply atlas/templates/Makefile.common.gotmpl]: completely removed this

[the remark this answers is not in the document]
Comment (kd7lxl) [reply atlas/templates/Makefile.common.gotmpl]: Thanks. This is going to cause other problems/dependencies in our build pipeline so we'll have to look into it.

**Review by kd7lxl — APPROVED** [review]

**Pull request merged**